app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @current_us = current_user
+    @comment = @current_us.comments.new(params.require(:comment).permit(:text))
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** As I mentioned in the previous comment, I think it would be better if you use the **"[the strong parameters](https://guides.rubyonrails.org/action_controller_overview.html#strong-parameters)"** like you studied in **[this lesson](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides)** for the project requirements

### NOTICE: 
I suggest you implemented all the changes marked as [ **OPTIONAL**] before merging your pull request for best practices